platform fsl p2020rdb patch

Add support for the Freescale P2020RDB-PCA platform.

diff --git a/arch/powerpc/boot/dts/p2020rdb.dts b/arch/powerpc/boot/dts/p2020rdb.dts
index 1d7a05f..7a1311d 100644
--- a/arch/powerpc/boot/dts/p2020rdb.dts
+++ b/arch/powerpc/boot/dts/p2020rdb.dts
@@ -16,9 +16,7 @@
 	compatible = "fsl,P2020RDB";
 
 	aliases {
-		ethernet0 = &enet0;
-		ethernet1 = &enet1;
-		ethernet2 = &enet2;
+		ethernet0 = &enet2;
 		serial0 = &serial0;
 		serial1 = &serial1;
 		pci0 = &pci0;
@@ -52,33 +50,35 @@
 				read-only;
 			};
 
-			partition@40000 {
-				/* 256KB for DTB Image */
-				reg = <0x00040000 0x00040000>;
-				label = "NOR (RO) DTB Image";
-				read-only;
+			partition@1 {
+				/* EEPROM data, 128KB, 1 sector */
+				reg = <0x00040000 0x00020000>;
+				label = "board_eeprom";
 			};
 
-			partition@80000 {
-				/* 3.5 MB for Linux Kernel Image */
-				reg = <0x00080000 0x00380000>;
-				label = "NOR (RO) Linux Kernel Image";
-				read-only;
+			partition@2 {
+				/* Available for apps */
+				reg = <0x00060000 0x00b00000>;
+				label = "open";
 			};
 
-			partition@400000 {
-				/* 11MB for JFFS2 based Root file System */
-				reg = <0x00400000 0x00b00000>;
-				label = "NOR (RW) JFFS2 Root File System";
+			partition@3 {
+				/* 4MB ONIE uImage */
+				reg = <0x00b60000 0x00400000>;
+				label = "onie";
 			};
 
-			partition@f00000 {
-				/* This location must not be altered  */
-				/* 512KB for u-boot Bootloader Image */
-				/* 512KB for u-boot Environment Variables */
-				reg = <0x00f00000 0x00100000>;
-				label = "NOR (RO) U-Boot Image";
-				read-only;
+			partition@4 {
+				/* U-Boot environment varialbes, 128KB, 1 sector */
+				reg = <0x00f60000 0x00020000>;
+				label = "uboot-env";
+				env_size = <0x2000>;
+			};
+
+			partition@5 {
+				/* 512KB u-boot */
+				reg = <0x00f80000 0x00080000>;
+				label = "uboot";
 			};
 		};
 
@@ -238,14 +238,11 @@
 		};
 
 		enet0: ethernet@24000 {
-			fixed-link = <1 1 1000 0 0>;
-			phy-connection-type = "rgmii-id";
+			status = "disabled";
 		};
 
 		enet1: ethernet@25000 {
-			tbi-handle = <&tbi0>;
-			phy-handle = <&phy0>;
-			phy-connection-type = "sgmii";
+			status = "disabled";
 		};
 
 		enet2: ethernet@26000 {
